doc/PLUGINS.md

```diff
@@ -314,6 +314,40 @@ It can return a non-zero `failure_code` field as defined for final
 nodes in [BOLT 4][bolt4-failure-codes], or otherwise an empty object
 to accept the payment.
 
+
+#### `openchannel`
+
+This hook is called whenever a remote peer tries to fund a channel to us,
+and it has passed basic sanity checks:
+
+```json
+{
+  "openchannel": {
+	"id": "03864ef025fde8fb587d989186ce6a4a186895ee44a926bfc370e2c366597a3f8f",
+    "funding_satoshis": "100000000msat",
+	"push_msat": "0msat",
+	"dust_limit_satoshis": "546000msat",
+	"max_htlc_value_in_flight_msat": "18446744073709551615msat",
+	"channel_reserve_satoshis": "1000000msat",
+	"htlc_minimum_msat": "0msat",
+	"feerate_per_kw": 7500
```

Comment:
missing comma end of line :-)

Reply:
Yes, and there's a trailing one on the last line, which isn't valid JSON either.  Fixed!